react/features/filmstrip/actions.web.js

```diff
@@ -282,17 +281,15 @@ export function setStageFilmstripViewDimensions() {
             width,
             columns,
             rows
-        } = disableResponsiveTiles
-            ? calculateNonResponsiveTileViewDimensions(state, true)
-            : calculateResponsiveTileViewDimensions({
-                clientWidth: availableWidth,
-                clientHeight,
-                disableTileEnlargement,
-                maxColumns,
-                noHorizontalContainerMargin: verticalWidth > 0,
-                numberOfParticipants,
-                numberOfVisibleTiles
-            });
+        } = calculateResponsiveTileViewDimensions({
+            clientWidth: availableWidth,
+            clientHeight,
+            disableTileEnlargement: false,
+            maxColumns,
```

Comment:
Probably you want to pass disableResponsiveTiles and disableTileEnlargement as `false` to the `getMaxColumns` function